dist/radar_client.js

```diff
@@ -31,7 +31,6 @@ module.exports = Backoff;
     Backoff = require('./backoff.js');
 
 instance._log = require('minilog');
-instance._logger = instance._log('radar_client');
```

Comment:
_log is referenced elsewhere.  _logger is not, and should not be, as it is specific to "radar_client".